web-ng/src/app/services/data-store/data-store.service.ts

```diff
@@ -55,6 +55,41 @@ export class DataStoreService {
       );
   }
 
+  /**
+   * Returns an Observable that loads and emits the projects to which the user
+   * is part of.
+   *
+   */
+  loadAccessibleProject$(userEmail: string): Observable<List<Project>> {
+    return this.db
+      .collection('projects', ref =>
+        ref.where(new firebase.firestore.FieldPath('acl', userEmail), 'in', [
+          'owner',
```

Comment:
Please move to constant, VALID_ROLES or similar.

Reply:
updated.